firebase-crashlytics/ktx/ktx.gradle

```diff
@@ -13,59 +13,43 @@
 // limitations under the License.
 
 plugins {
-    id 'firebase-library'
-    id 'kotlin-android'
+  id "firebase-library"
+  id "kotlin-android"
 }
 
 firebaseLibrary {
-    libraryGroup "crashlytics"
-    testLab.enabled = true
-    publishJavadoc = true
-    publishSources = true
+  libraryGroup "crashlytics"
+  testLab.enabled = true
+  publishJavadoc = true
+  publishSources = true
 }
 
 android {
-    compileSdkVersion 33
-    defaultConfig {
-        minSdk 16
-        multiDexEnabled true
-        targetSdk 33
-        versionName version
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-    }
-    sourceSets {
-        main.java.srcDirs += 'src/main/kotlin'
-        test.java {
-            srcDir 'src/test/kotlin'
-        }
-        androidTest.java.srcDirs += 'src/androidTest/kotlin'
-    }
-    testOptions.unitTests.includeAndroidResources = true
+  compileSdk 33
+  defaultConfig {
+    minSdk 16
+    multiDexEnabled true
+    targetSdk 33
+    testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+  }
+  compileOptions {
+    sourceCompatibility JavaVersion.VERSION_1_8
+    targetCompatibility JavaVersion.VERSION_1_8
+  }
+  kotlinOptions {
+    jvmTarget = '1.8'
+  }
+  testOptions.unitTests.includeAndroidResources = true
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
-
-    implementation 'com.google.firebase:firebase-common:20.3.1'
-    implementation 'com.google.firebase:firebase-components:17.1.0'
-    implementation 'com.google.firebase:firebase-common-ktx:20.3.1'
-    implementation project(':firebase-crashlytics')
-    implementation 'androidx.annotation:annotation:1.1.0'
-
-    testImplementation "org.robolectric:robolectric:$robolectricVersion"
-    testImplementation 'junit:junit:4.12'
-    testImplementation "com.google.truth:truth:$googleTruthVersion"
-    testImplementation 'org.mockito:mockito-core:2.25.0'
-
-    androidTestImplementation 'junit:junit:4.12'
-    androidTestImplementation "com.google.truth:truth:$googleTruthVersion"
-    androidTestImplementation 'androidx.test:runner:1.2.0'
-    androidTestImplementation 'androidx.test:core:1.2.0'
+  implementation "com.google.firebase:firebase-common-ktx:20.3.2"
```

Comment:
Seems like we have pruned a lot of dependencies. Can we ensure that there are no functionality breakage here? I'm guessing many of these changes need to be directly done in the `master` branch?

Reply:
Yeah I think this build file was originally copied from the perf ktx build file, and a lot of the deps that were copied were never used. For example, all the testImplementation deps are not needed since there are no unit tests, only android tests.